### PR TITLE
Feature/role based shiksha sathi users

### PR DESCRIPTION
**Pull request by Coding-Thunder**
1. Created user service class util to perform the basic user actions 
2. Bound Shiksha Sathi users to their relative geographic level based on their role

### CODE REVIEW

File: admin/utils/interfaces.ts
Comment (amit-s19) [inline admin/utils/interfaces.ts:1]: district spelling is not right.

File: admin/utils/user.util.ts
Comment (amit-s19) [inline admin/utils/user.util.ts:1]: Please fix district spelling in constructor. 
Please compare on applicationId. It's not necessary that first registration would be of Shiksha Saathi

**Review by amit-s19 — CHANGES_REQUESTED** [review]
Please make this required changes

File: admin/utils/user.util.ts
Comment (Coding-Thunder) [reply admin/utils/user.util.ts:1]: by iterating into the registrations ?

**Review by amit-s19 — APPROVED** [review]
LGTM

**Pull request merged**